src/gossip/service.zig

```diff
@@ -2616,10 +2616,8 @@ test "gossip.gossip_service: test packet verification" {
     const id = Pubkey.fromPublicKey(&keypair.public_key);
     const contact_info = try localhostTestContactInfo(id);
 
-    var logger = Logger.init(std.testing.allocator, Logger.TEST_DEFAULT_LEVEL);
-    defer logger.deinit();
-    logger.spawn();
-
+    // noop for this case because this tests error failed verification
```

Comment:
I'm a little confused by this comment. Does this mean that error messages are expected because this is a negative test? If so, I agree with making the logger a noop.

Reply:
yup exactly